### ISSUE TITLE
[Accessbility]A11y_Java Tooling for IntelliJ/Eclipse - New Feature_Link a Cluster-Link a Cluster_HD Insight_Keyboard: Focus is not visible on "Help" control in IntelliJ theme

### ISSUE THREAD

**Issue by Amulya-95** [microsoft/azure-tools-for-java#3627]
[Check out Accessibility Insights!](https://github.com/microsoft/azure-tools-for-java/issues/url) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHub Tags: #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #A11yMAS; #A11yTCS; #C+E-H; #DesktopApp; #Win10; #JavatoolingforIntelliJ/Eclipse; #Keyboard; #MAS2.4.7

**Environment Details:**
Download and Install IntelliJ IDEA from https://www.jetbrains.com/idea/download/#section=windows
IntelliJ IDEA 2019.2.2
Build #IU-192.6603.28

**OS Details:**
OS: Windows (Version 1903, Build: 18362.356)
MAS Violated: MAS2.4.7

**Repro Steps:**
1)Launch IntelliJ IDEA in IntelliJ theme
2)Tab till Create New Project button and press enter
3)Tab till view-->Tool Window-->Azure Explorer and press enter
4)Tab till signin-->copy and open & sign in to azure
5)Tab till select button and press enter
6)Tab till HDInsight and open context menu by using keyboard key
7)Tab till link a cluster and press enter
8)Tab till 'Help' control and check whether focus is visible on the control

**Actual Result:**
Focus is not visible on 'Help' control in IntelliJ theme

**Expected Result:**
Focus should be visible on 'Help' control in IntelliJ theme

**User Impact:**
Keyboard only users will find difficulty if the focus is not visible on the controls

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}

![Focus on 'Help' control is nopt visible in light theme](https://user-images.githubusercontent.com/51435266/65574467-34c0a780-df8b-11e9-925c-2f33577db2e7.png)

**Comment by t-rufang** [comment]
This dialog is extended from class `com.intellij.openapi.ui.DialogWrapper`, which is supported by IntelliJ and it's out of the scope of our toolkit.
Please find another same issue in IntelliJ. As you can see, focus in help button in light theme is not very obvious. 
![issue_3627_lighttheme](https://user-images.githubusercontent.com/32627233/68269472-e8da4580-0094-11ea-853f-080e3628db34.gif)

![issue_3627_darktheme](https://user-images.githubusercontent.com/32627233/68269407-aadd2180-0094-11ea-9049-f291a8e685fb.gif)

**Comment by t-rufang** [comment]
- Importance of experience: Minimal. Focus is invisible on 'Help' control in IntelliJ theme.
- Quality of experience: managable. Keyboard user can change to Dracula theme or High contrast theme in which focus is visible.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue.
- [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by Alexandria-R** [comment]
Approved to close as external to intelliJ

**Comment by Amulya-95** [comment]
Closing this bug as per the discussion with SME

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749725
